easybuild/easyconfigs/p/Python/Python-2.7.13-foss-2016b-bare.eb

```diff
@@ -0,0 +1,147 @@
+name = 'Python'
+version = '2.7.13'
+versionsuffix = '-bare'
+
+homepage = 'http://python.org/'
+description = """Python is a programming language that lets you work more quickly and integrate your systems
+ more effectively.
+
+ This is a version without Tk or X11."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+
+# python needs bzip2 to build the bz2 package
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('zlib', '1.2.8'),
+    ('libreadline', '6.3'),
+    ('ncurses', '6.0'),
+    ('SQLite', '3.13.0'),
+    ('GMP', '6.1.2'),
+    ('libffi', '3.2.1'),
+    # OS dependency should be preferred if the os version is more recent then this version,
+    # it's nice to have an up to date openssl for security reasons
+    # ('OpenSSL', '1.0.2h'),
+]
+
+osdependencies = [('openssl-devel', 'libssl-dev', 'libopenssl-devel')]
+
+# order is important!
+# package versions updated 9th February 2017
+# using easy_update
+# the commented out extensions were added by easy_update but broke the build
+exts_list = [   
+    ('six', '1.10.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/s/six/'],
+    }),
+    #('packaging', '16.8', {
+    #    'source_urls': ['https://pypi.python.org/packages/source/p/packaging/'],
+    #}),
+    #('pyparsing', '2.1.10', {
+    #    'source_urls': ['https://pypi.python.org/packages/source/p/pyparsing/'],
+    #}),
+    #('appdirs', '1.4.0', {
+    #    'source_urls': ['https://pypi.python.org/packages/source/a/appdirs/'],
+    #}),
+    ('setuptools', '32.3.1', {
+        'source_tmpl': '%(name)s-%(version)s.zip',
+        'source_urls': ['https://pypi.python.org/packages/source/s/setuptools/'],
+    }),
+    ('pip', '9.0.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/pip/'],
+    }),
+    ('nose', '1.3.7', {
+        'source_urls': ['https://pypi.python.org/packages/source/n/nose/'],
+    }),
+    ('numpy', '1.12.0', {
+        'source_tmpl': '%(name)s-%(version)s.zip',
+        'source_urls': ['https://pypi.python.org/packages/source/n/numpy'],
+    }),
```

Comment:
@wpoely86 I think we should trash the `-bare` easyconfig, since it's not bare at all with `numpy` included...

I think @verdurin had this to play around with a Python with/without Tkinter, but it's now no longer relevant...